framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -84,6 +96,7 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 	private _sidechainAPIClient!: apiClient.APIClient;
 	private _chainConnectorState!: ChainConnectorInfo;
 	private readonly _sentCCUs: SentCCUs = [];
+	private _privateKey!: Buffer;
```

Comment:
This will be `EncryptedMessageObject` check `encryptMessageWithPassword` in `lisk-cryptography`